Mathlib/CategoryTheory/Extensive.lean

```diff
@@ -287,6 +287,7 @@ instance types.finitaryExtensive : FinitaryExtensive (Type u) := by
 
 section TopCat
 
+set_option maxHeartbeats 250000 in
```

Comment:
This is `continuity`'s fault.

Reply:
@erdOne's new proof avoids it beautifully!